firefox_decrypt.py

```diff
@@ -746,14 +752,19 @@ def parse_sys_args():
     parser = argparse.ArgumentParser(
         description="Access Firefox/Thunderbird profiles and decrypt existing passwords"
     )
-    parser.add_argument("profile", nargs='?', default=profile_path,
+    parser.add_argument("profile", nargs="?", default=profile_path,
                         help="Path to profile folder (default: {0})".format(profile_path))
     parser.add_argument("-e", "--export-pass", action="store_true",
                         help="Export URL, username and password to pass from passwordstore.org")
     parser.add_argument("-p", "--pass-prefix", action="store", default=u"web",
                         help="Prefix for export to pass from passwordstore.org (default: %(default)s)")
-    parser.add_argument("-t", "--tabular", action="store_true",
-                        help="Output in tabular format")
+    parser.add_argument("-f", "--format", action="store", choices={"csv", "human"},
+                        default="human", help="Format for the output.")
+    parser.add_argument("-d", "--delimiter", action="store", default=";",
+                        help="The delimiter for csv output")
+    parser.add_argument("-q", "--quotechar", action="store", default="|",
```

Comment:
Can you set this default to `"` instead of `|` to use slightly more common CSV settings?